app/controllers/actions/DataRequiredAction.scala

```diff
@@ -29,7 +29,7 @@ class DataRequiredActionImpl @Inject() (implicit val executionContext: Execution
   override protected def refine[A](request: OptionalDataRequest[A]): Future[Either[Result, DataRequest[A]]] =
     request.userAnswers match {
       case None =>
-        Future.successful(Left(Redirect(routes.JourneyRecoveryController.onPageLoad())))
+        Future.successful(Left(Redirect(routes.ThereIsAProblemController.onPageLoad())))
```

Comment:
@iservport Instead of ThereIsAProblem here we can call SessionExpiresController.onPageLoad